.travis.yml

```diff
@@ -46,12 +53,28 @@ install:
 
 # Run test
 script:
+  # test docs
   - nosetests $TEST_DIR --with-cov --cov SimPEG --cov-config .coveragerc -v -s
 
 # Calculate coverage
 after_success:
   - coveralls --config_file .coveragerc
 
+  - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
```

Comment:
we should do this in a makefile instead (it would clean up the travis file a lot)